python/ray/data/tests/test_operators.py

```diff
@@ -226,6 +227,14 @@ def _sleep(block_iter: Iterable[Block]) -> Iterable[Block]:
     wait_for_condition(lambda: (ray.available_resources().get("GPU", 0) == 1.0))
 
 
+def test_prograss_bar():
```

Comment:
why this unit test is testing progress bar? didn't see any assertion around progress bar, so asking.

Reply:
It's a bit indirect because the progress bar is a local variable that's involved in execution. Make it test num outputs for operators.